study.md

```diff
@@ -38,29 +38,65 @@ As it makes sense to you please write the following in JSON format:
 - Has lived in two places recently: 123 Fake street, and 2 Muchinfo Road
 
 ```json
-<!-- your answer here -->
-```
+{"menu": {
+  "id" : "Jason",
+  "value" : "possesions",
+  "cat": "Mr. Kitty",
+  "roomates": [
```

Comment:
why did you chose do do an array of objects there.  It's not wrong, just curious